.github/workflows/preview-build.yml

```
@@ -438,3 +444,58 @@ jobs:
                 body:body.join('\n'),
               });
             }
+      - name: Comment on docs changes about versioning requirements
+        if: inputs.enable-cumulative-comment == 'true'
```

```suggestion
        if: inputs.enable-cumulative-comment == 'true'
        && needs.check.outputs.any_modified == 'true'
```

@Mpdreamz I'm sure it's not as simple as that 😆 

Had forgotten about this again until I looked at my TODOs, would be nice to get this change in for elasticsearch tho

---

The whole job already has the condition to only trigger on changes

---

doh missed that!